sematic/ui/packages/common/src/component/TagsList.tsx

```diff
@@ -0,0 +1,34 @@
+import Chip, { chipClasses } from '@mui/material/Chip';
+import Box from '@mui/material/Box';
+import Button, { buttonClasses } from '@mui/material/Button';
+import styled from '@emotion/styled';
+import theme from 'src/theme/new';
+
+const StyledBox = styled(Box)`
+    display: flex;
+    align-items: center;
+
+    & .${chipClasses.root}:not(:first-child) {
+        margin-left: ${theme.spacing(1)};
+    }
+
+    & .${buttonClasses.root} {
+        margin-left: ${theme.spacing(3)};
+        padding: ${theme.spacing(1)};
+        height: 25px;
+    }
+`;
+
+interface TagsListProps {
+    tags: string[];
+}
+
+const TagsList = (props: TagsListProps) => {
+    const { tags = [] } = props;
+    return <StyledBox>
+        {tags.map(tag => <Chip key={tag} label={tag} />)}
+        <Button variant={"text"}>add tags</Button>
```

Comment:
Shouldn't this be blue, or styled differently? Now it looks greyed-out.

Reply:
The button will turn blue when hovered. It is not always grey.

We can use this page to inspect that behavior interactively.
https://story.dev-usw2-sematic0.sematic.cloud/?path=/story/sematic-page--page